The NixOS test fails for me with
```
server # [   43.696682] vaultwarden[811]: [2024-04-08 19:37:20.956][request][INFO] GET /api/config
server # [   43.699462] vaultwarden[811]: [2024-04-08 19:37:20.956][response][INFO] (config) GET /api/config => 200 OK
server # [   53.816437] test-runner[927]: Traceback (most recent call last):
server # [   53.822889] test-runner[927]:   File "/run/current-system/sw/bin/test-runner", line 18, in <module>
server # [   53.832460] test-runner[927]:     wait.until(EC.title_contains("Create account"))
server # [   53.840907] test-runner[927]:   File "/nix/store/bk2kan47zhbf6zzka099wc0s2dp66car-python3-3.11.8-env/lib/python3.11/site-packages/selenium/webdriver/support/wait.py", line 105, in until
server # [   53.850198] test-runner[927]:     raise TimeoutException(message, screen, stacktrace)
server # [   53.852376] test-runner[927]: selenium.common.exceptions.TimeoutException: Message:
```

--- a/nixos/tests/vaultwarden.nix
+++ b/nixos/tests/vaultwarden.nix
@@ -106,7 +106,7 @@ let
 
                   wait = WebDriverWait(driver, 10)
 
-                  wait.until(EC.title_contains("Create account"))
+                  wait.until(EC.title_contains("Vaultwarden Web"))
 
                   driver.find_element(By.CSS_SELECTOR, 'input#register-form_input_email').send_keys(
                       '${userEmail}'

